mappings/net/minecraft/entity/passive/HorseBaseEntity.mapping

```diff
@@ -21,6 +21,8 @@ CLASS net/minecraft/class_1496 net/minecraft/entity/passive/HorseBaseEntity
 	FIELD field_6975 soundTicks I
 	FIELD field_6976 jumpStrength F
 	METHOD method_26899 createBaseHorseAttributes ()Lnet/minecraft/class_5132$class_5133;
+	METHOD method_27930 findPassengerDismountLocationInternal (Lnet/minecraft/class_243;Lnet/minecraft/class_1309;)Lnet/minecraft/class_243;
```

Comment:
```suggestion
	METHOD method_27930 findDismountLocationInternal (Lnet/minecraft/class_243;Lnet/minecraft/class_1309;)Lnet/minecraft/class_243;
```

Reply:
I agree, only a passenger would be dismounting.